meetupHelpers.js

```diff
@@ -1,5 +1,29 @@
-import { joinArrayHumanReadable } from './utils.js';
 import fetch from 'node-fetch';
+import { joinArrayHumanReadable } from './utils.js';
+
+export async function fetchTodaysEvent () {
+  const nextEvent = await fetchNextEventForGroup(34547654)
+  
+  const eventDate = nextEvent.dateTime.split('T')[0]
+  
+  const now = new Date();
+  
+  const todayArray = [now.getFullYear(), now.getMonth() + 1, now.getDate()]
+  
+  if (todayArray[1] < 10) {
+    todayArray[1] = `0${todayArray[1]}`
+  }
+  
+  if (todayArray[2] < 10) {
+    todayArray[2] = `0${todayArray[2]}`
+  }
+  
+  if (eventDate !== todayArray.join('-')) {
+    return null
+  }
```

Comment:
This technically works, but https://date-fns.org/v2.28.0/docs/isToday is preferable. Then it would be: 
```
return isToday(eventDate) ? nextEvent : null;
```